containers/conclave-build/Dockerfile

```diff
@@ -11,23 +11,24 @@ ENV LC_ALL en_US.UTF-8
 
 # TODO curl and apt-transport-https are only needed to install gramine, so they should be run as part of a single
 #  command and then removed.
+# Disable interactive dialogue
+# Otherwise, a message that requires user input will appear while installing Java 17
 RUN apt-get -y update && DEBIAN_FRONTEND=noninteractive apt-get install -y \
         build-essential \
         bash \
         curl \
         apt-transport-https \
-        python3-pip
+        python3-pip \
+        openjdk-17-jdk-headless
```

Comment:
Not sure why this was actually after adding the Gramine repository, I feel it is nicer to separate the installation after adding the Gramine repository.

Reply:
Move the setting of JAVA_HOME next to this.

Reply:
I also moved the CMD comand closer

Reply:
I don't think the java -version cmd is necesary

Reply:
Removing